scss/_variables.scss

```diff
@@ -77,9 +77,12 @@ $swatch-7-dark: $peach-dark !default;
 
 /* SJC Button Overrides ***************************************/
 
-$btn-border-radius: 1.55rem !default;
-$btn-border-radius-lg: 1.55rem !default;
-$btn-border-radius-sm: 1.55rem !default;
-$btn-padding-y: .94rem !default;
-$btn-padding-x: 1.9rem !default;
-$btn-line-height: .5 !default;
+$btn-border-radius: 5rem !default;
+$btn-border-radius-lg: 5rem !default;
+$btn-border-radius-sm: 5rem !default;
```

Comment:
This looks better, IMO, but the style guide states 25px rounded.

Reply:
Isn't that how they achieved the fully rounded ends since theirs were ~40px tall? My understanding then is that this would keep that look in the case a button accidentally goes to two lines, correct?

Reply:
You are probably right. I just wanted to point it out as something that will likely need to be revised in the style guide.

Reply:
Style guide or brand guidelines?

Reply:
Brand Reference Guide:

> With the exception of the research domains, all buttons and other action items appear in HEX #1874dc. The buttons are 40 pixels tall x the appropriate width to accommodate text with rounded corners (25px rounded). The buttons will lighten to an 85% tint of HEX #1874dc when hovered. The padding around all buttons should be no less than 30 pixels.

We've already established that we are going more with the spirit of the brand guide for now, so I'm guessing it's not an issue.

Reply:
Great! I'll take note for updating the brand guidelines.